packages/form-utils/src/lib/Select.stories.tsx

```diff
@@ -64,7 +65,19 @@ export const _SelectPlaceholder: StoryObj<typeof Select> = {
       </FormControl>
     );
   },
-  args: { label: 'count', id: 'count', size: 'small', placeholder: 'Select...' },
+  args: {
+    label: 'count',
+    id: 'count',
+    size: 'small',
+    displayEmpty: true,
+    renderValue: (selected: string) => {
+      if (!selected) {
+        return <Typography color="grey.400">Placeholder</Typography>;
```

Comment:
can we update the color to `secondary.main` (same color), or double check with UX that this shouldn't be using `text.secondary`(slightly darker).

Can't let people get the idea they can just pick colors lol

Reply:
replaced with the new text.placeholder token